<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_statistics</class>
 <widget class="QWidget" name="main_statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Начальный период:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="date1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Конечный период:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="date2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Дисциплина:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboSubject">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Группа:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboGroup">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Тест:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboTest">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Преподаватель:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboPrepod">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="table" name="tableStat"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>table</class>
   <extends>QTableWidget</extends>
   <header>Widgets/table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
